src/App.js

```diff
@@ -1,12 +1,81 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import Pokemon from "./Pokemon";
+import axios from 'axios';
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+
+    const [pokenames, setPokenames] = useState(null);
+    const [offset, setOffset] = useState(0);
+
+
+    function handleClickNext() {
+        setOffset(offset + 20)
+        setPokenames([])
+    };
+
+    function handleClickPrevious() {
+        setOffset(offset - 20)
+        setPokenames([])
+    };
+
+    useEffect(() => {
+        async function fetchPokenames() {
+            try {
+                const result = await
+                    axios.get(`https://pokeapi.co/api/v2/pokemon/?limit=20&offset=${offset}`);
+                setPokenames(result.data.results);
+                console.log("what is inside pokenames", result.data.results);
+            } catch (e) {
+                console.error(e);
+
+            }
+        }
+
+        fetchPokenames();
+
+
+    }, [offset]);
+
+
+    return (
+        <>
+            <div className="cardContainerMain">
+                <img className="logo"
+                     src="https://upload.wikimedia.org/wikipedia/commons/9/98/International_Pok%C3%A9mon_logo.svg"/>
+            </div>
+            <div className="buttonContainer">
+                <button
+                    className="navButtons"
+                    disabled={!offset}
+                    onClick={() => handleClickPrevious()}
+                >
+                    vorige
+                </button>
+
+                <button
+                    className="navButtons"
+                    onClick={() => handleClickNext()}
+                >
+                    volgende
+                </button>
+            </div>
+
+            <div>
+
+                {pokenames ? (<div className="cards">{pokenames.map((pokename) => {
+                        return <Pokemon nameOfPokemon={pokename.name}/>;
+                    })}
+                    </div>
+                ) : (
+                    <h3> Loading... </h3>
```

Comment:
Als er de eerste keer een error optreedt dan blijft er voor altijd dit staan. Ik zou liever zien dat je in de useEffect een state `loading` op true zet o.i.d. Dan wordt het ook iets meer expliciet